packages/theme/README.md

```diff
@@ -55,10 +67,40 @@ The above import will get you started using the `<sp-theme>` wrapper element, an
 
 Once you've moved beyond the prototype phases of an application, it is likely that you will only use one combinatin of `color` and `scale` in your application, and even when you don't you will likely benefit from lazily loading variants that you don't leverage by default. For single combination applications or to power a _default_ theme, the following imports can be used to ensure only the code your application requires is loaded:
 
+### Classic
+
 ```js
-// Power a site using <sp-theme color="darkest" scale="large">
+/**
+ * Power a site using
+ *
+ * <sp-theme
+ *      flavor="classic"
+ *      color="darkest"
+ *      scale="large"
+ * >
+ **/
 import '@spectrum-web-components/theme/theme-darkest.js';
 import '@spectrum-web-components/theme/scale-large.js';
+import '@spectrum-web-components/theme/flavor-classic.js';
+
+import '@spectrum-web-components/theme/sp-theme.js';
+```
+
+### Express
+
+```js
+/**
+ * Power a site using
+ *
+ * <sp-theme
+ *      flavor="express"
+ *      color="light"
+ *      scale="medium"
+ * >
+ **/
+import '@spectrum-web-components/theme/express/theme-light.js';
```

Comment:
Made this import come along with the color/scale imports by default and evacuated the "spectrum" theme registrations from `Theme.ts` with the same technique so that the two systems can be more fully separate (reducing JS payload for Express).